ztp/ran-site-plan-crd/site-config-crd.yaml

```diff
@@ -0,0 +1,298 @@
+                  type: string
+                pullSecret:
+                  description: PullSecret to use when pulling images.
+                  type: string
+                releaseImageUrl:
+                  description: |
+                    releaseImageUrl is the url to the image containing metadata for all
+                    components that run in the cluster, and is the primary and best way
+                    to specify what specific version of OpenShift you wish to install.
+                  type: string
+                sshPublicKey:
+                  description: sshPublicKey will be added to all cluster hosts for use in debugging.
+                  type: string
+                sshPrivateKey:
+                  description: |
+                    sshPrivateKey is the private SSH key to use for access to
+                    compute instances. This private key should correspond to
+                    the public key included in the sshPublicKey. The
+                    private key is used by Hive to gather logs on the target
+                    cluster if there are install failures. The SSH private
+                    key is expected to be in the secret data under the
+                    "ssh-privatekey" key.
+                  type: string
+                registry:
+                  description: contain the required config for the mirroring registry
+                  type: object
+                  properties:
+                    registries.conf:
+                      description: registryConf holds the content of registries.conf file configured with mirror registries
+                      type: string
+                    ca-bundle.crt:
+                      description: registryCert hold the contents of mirror registry certificate/s
+                      type: string
+                clusters:
+                  description: List of clusters tha will be installed in the site.
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      clusterName:
+                        description: |
+                          ClusterName is the friendly name of the cluster. It is used
+                          for subdomains, some resource tagging, and other instances
+                          where a friendly name for the cluster is useful.
+                        type: string
+                      baseDomain:
+                        description: BaseDomain is the base domain to which the cluster should belong.
+                        type: string
+                      clusterProfile:
+                        description: |
+                          clusterProfile speicfy the day0 configuration that will get appllied on the cluster must be one
+                          of the following du-sno, du, cu, cu-du
+                        type: string
```

Comment:
We are combining two things here that should be separated.
1) Cluster type - standard or sno
2) Role - du/cu 

What is a cu-du ?

Reply:
from config view: 
du-sno -> to use the master node label for policy match config
du -> to add MCP du then use node label for policy match config
cu -> to add MCP cu then node label for policy match config
cu-du -> to add MCPs cu and du; early we were discussing that in LDC one cluster could manage CU and DU

Reply:
I'd vote to only include the values we know we need for now, until we define other profile types.

Reply:
To @browsell 's comment, should we split this into:

clusterType: [sno|standard]
clusterProfile: [du|cu|none|...]

Reply:
done. separate them